### PR TITLE
docs: add pull request section to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by piperchester**
## Motivation and Context
This is something I ran into when beginning work on https://github.com/lerna/lerna/issues/2198. I'm happy to update the docs to only allow forking if that's the contribution model we'd rather see here, but wanted to start the discussion.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -106,3 +106,7 @@ $ open coverage/lcov-report/index.html
 # Linux
 $ xdg-open coverage/lcov-report/index.html
 ```
+
+### Submitting Pull Requests
+
+If you've cloned the project directly and are working on a feature branch, you'll need write permissions to be able to push that branch to the remote to open a pull request.
```
Comment (dizel3d) [inline CONTRIBUTING.md]: I don't see a reason to share write permission with contributors. Fork is only right way.

**Comment by piperchester** [comment]
Sounds good! Thanks @3imed-jaberi @dizel3d @evocateur. I cam amend the PR to encourage forking.

File: CONTRIBUTING.md (hunk 1 above)
Comment (piperchester) [inline CONTRIBUTING.md]:
```suggestion
This project follows GitHub's standard forking model. Please fork the project to submit pull requests. 
```

**Comment by dizel3d** [comment]
@piperchester you may also add the link to respective [github guide](https://guides.github.com/activities/forking)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -106,3 +106,7 @@ $ open coverage/lcov-report/index.html
 # Linux
 $ xdg-open coverage/lcov-report/index.html
 ```
+
+### Submitting Pull Requests
+
+This project follows GitHub's standard forking model. Please fork the project to submit pull requests.
```
Comment (piperchester) [inline CONTRIBUTING.md]:
```suggestion
This project follows [GitHub's standard forking model](https://guides.github.com/activities/forking/). Please fork the project to submit pull requests. 
```

**Pull request merged**